### ISSUE TITLE
Unable to get property 'WebServiceProxy' of undefined or null reference

### ISSUE THREAD

**Issue by eldridgegl** [DevExpress/AjaxControlToolkit#164]
Running into an issue when attempting to use an ajaxcontroltoolkit extension. This was a website converted to web application. The extension worked as a website but after the conversion to web application, it no longer works. Things have change since then as seen below attempting to get this working again. Any help would be appreciated.  AjaxControlToolkit V16.1. The runtime error 0x800a138f - JavaScript runtime error: Unable to get property 'WebServiceProxy' of undefined or null reference.  the creation of the dynamic page reveals the issue at this point:

```
GetCompletionList:function(prefixText,count,contextKey,succeededCallback, failedCallback, userContext) {
return this._invoke(this._get_path(), 'GetCompletionList',false,{prefixText:prefixText,count:count,contextKey:contextKey},succeededCallback,failedCallback,userContext); },
GetPartsAndDescriptions:function(prefixText,count,succeededCallback, failedCallback, userContext) {
return this._invoke(this._get_path(), 'GetPartsAndDescriptions',false,{prefixText:prefixText,count:count},succeededCallback,failedCallback,userContext); }}
PageMethods.registerClass('PageMethods',Sys.Net.WebServiceProxy);
```
The registered assembly:
```
<%@ Page Title="Purchasing Manager" Language="C#" MasterPageFile="~/Site.master" AutoEventWireup="true" Inherits="PurchasingManager" CodeBehind="PurchasingManager.aspx.cs" %>
<%@ Register Assembly="AjaxControlToolkit" Namespace="AjaxControlToolkit" TagPrefix="ajaxToolkit" %>
```
I am using version 16.1 ajaxcontroltoolkit. The script manager is setup as so:
```
   <asp:ScriptManager ID="ScriptManager1" runat="server" EnablePartialRendering="true"
        EnablePageMethods="true" LoadScriptsBeforeUI="False" ScriptMode="Release" AjaxFrameworkMode="Explicit">
        <CompositeScript>
            <Scripts>
                <asp:ScriptReference Path="~/Scripts/MyScripts.js"  />
            </Scripts>
        </CompositeScript>

    </asp:ScriptManager>
```
The ajaxtoolkit:autocompleteextender part of the code for textbox tb_filter is as follows:
```
    <div id="SearchControl" runat="server">
        <asp:Label ID="Label3" runat="server" Font-Size="12pt" Text="Filter Search:" Width="200px"
            Font-Bold="True"></asp:Label>
        <asp:TextBox ID="tb_Filter" runat="server" autocomplete="off" AutoPostBack="true" OnTextChanged="txtSearch_TextChanged" onkeyup="return true;" ClientIDMode="Static"
            Width="245px"></asp:TextBox>
        <script type="text/javascript">
            function clearTextBox() {
                document.getElementById('<%= tb_Filter.ClientID %>').value = "";
            }
        </script>
        <asp:LinkButton ID="lb_clearText" runat="server" Font-Size="12pt"
            OnClientClick="clearTextBox()"><--Clear Text</asp:LinkButton>
    </div>
    <ajaxToolkit:AutoCompleteExtender ID="tb_Filter_AutoCompleteExtender" runat="server"
        CompletionInterval="220" CompletionSetCount="1" CompletionListCssClass="AutoCompleteExtender_CompletionList"
        CompletionListHighlightedItemCssClass="AutoCompleteExtender_HighlightedItem"
        CompletionListItemCssClass="AAutoCompleteExtender_CompletionListItem" DelimiterCharacters=""
        Enabled="True" MinimumPrefixLength="1" ServiceMethod="GetPartsAndDescriptions"
        TargetControlID="tb_Filter">
    </ajaxToolkit:AutoCompleteExtender>
```
The referenced servicemethod for codebehind is:
```
 [System.Web.Services.WebMethodAttribute(), System.Web.Script.Services.ScriptMethodAttribute()]
        public static string[] GetPartsAndDescriptions(string prefixText, int count)
        {
            prefixText = prefixText.ToLower();

            int length = prefixText.Length;
            string ConcatItem = string.Empty;
            List<string> retList = new List<string>();
            try
            {
                DataTable dt = DataAccess.PartsList.DataAccesser.GetAllItemsWithAllFields("Active");
                foreach (DataRow row in dt.Rows)
                {
                    bool AddToList = false;
                    string EPIPartNo = row["EPIPartNo"].ToString();
                    string Description = row["Description"].ToString();

                    if (length <= EPIPartNo.Length)
                        if (EPIPartNo.Substring(0, length).ToLower().Equals(prefixText))
                            AddToList = true;
                    if (length <= Description.Length)
                        if (Description.Substring(0, length).ToLower().Equals(prefixText))
                            AddToList = true;

                    if (AddToList)
                    {
                        string retString = string.Format("{0}| DESC: {1}", EPIPartNo, Description);
                        retList.Add(retString);
                    }
                }
            }
            catch (System.Exception ex)
            {
                Helper.EventLogConstants.WriteEventLog(
                              CONSTS.CONSTS.EPISERV_APPLICATION_NAME + "\n" +
                              string.Format("{0}: {1}\n", "Method", System.Reflection.MethodBase.GetCurrentMethod().Name) +
                              ex.Message + ex.StackTrace,
                              System.Diagnostics.EventLogEntryType.Error);
            }
            return retList.ToArray();
        }
```
The web.config file is as follows:
```
<?xml version="1.0"?>
<!--
  For more information on how to configure your ASP.NET application, please visit
  http://go.microsoft.com/fwlink/?LinkId=169433
  -->
<configuration>
  <configSections>
    <sectionGroup name="devExpress">
      <section name="themes" type="DevExpress.Web.ThemesConfigurationSection, DevExpress.Web.v14.2, Version=14.2.13.0, Culture=neutral, PublicKeyToken=b88d1754d700e49a" requirePermission="false"/>
      <section name="settings" type="DevExpress.Web.SettingsConfigurationSection, DevExpress.Web.v14.2, Version=14.2.13.0, Culture=neutral, PublicKeyToken=b88d1754d700e49a" requirePermission="false"/>
      <section name="errors" type="DevExpress.Web.ErrorsConfigurationSection, DevExpress.Web.v14.2, Version=14.2.13.0, Culture=neutral, PublicKeyToken=b88d1754d700e49a" requirePermission="false"/>
      <!--<section name="compression" type="DevExpress.Web.ASPxClasses.CompressionConfigurationSection, DevExpress.Web.v13.2, Version=13.2.5.0, Culture=neutral, PublicKeyToken=b88d1754d700e49a" requirePermission="false"/>-->
      <section name="compression" type="DevExpress.Web.CompressionConfigurationSection, DevExpress.Web.v14.2, Version=14.2.13.0, Culture=neutral, PublicKeyToken=b88d1754d700e49a" requirePermission="false"/>
    </sectionGroup>
    <!-- For more information on Entity Framework configuration, visit http://go.microsoft.com/fwlink/?LinkID=237468 -->
    <section name="entityFramework" type="System.Data.Entity.Internal.ConfigFile.EntityFrameworkSection, EntityFramework, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" requirePermission="false"/>
    <!-- For more information on Entity Framework configuration, visit http://go.microsoft.com/fwlink/?LinkID=237468 -->
    <section name="doddleReport" type="DoddleReport.Configuration.DoddleReportSection, DoddleReport"/>

    <sectionGroup name="system.web.extensions" type="System.Web.Configuration.SystemWebExtensionsSectionGroup, System.Web.Extensions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35">
      <sectionGroup name="scripting" type="System.Web.Configuration.ScriptingSectionGroup, System.Web.Extensions,Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35">
        <section name="scriptResourceHandler" type="System.Web.Configuration.ScriptingScriptResourceHandlerSection, System.Web.Extensions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" requirePermission="false" allowDefinition="MachineToApplication"/>
        <sectionGroup name="webServices" type="System.Web.Configuration.ScriptingWebServicesSectionGroup, System.Web.Extensions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35">
          <section name="jsonSerialization" type="System.Web.Configuration.ScriptingJsonSerializationSection, System.Web.Extensions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" requirePermission="false" allowDefinition="Everywhere"/>
          <section name="profileService" type="System.Web.Configuration.ScriptingProfileServiceSection, System.Web.Extensions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" requirePermission="false" allowDefinition="MachineToApplication"/>
        </sectionGroup>
      </sectionGroup>
    </sectionGroup>
  </configSections>

  <connectionStrings>
    <add name="MasterDatabaseConnectionString" connectionString="data source=C:\Users\Test User\Desktop\newEPIWebAppSolution\EPIBiz\EPIBiz\MasterDatabase.db3;Version=3;Journal Mode=WAL;" providerName="System.Data.SQLite"/>
    <add name="UserDataConnection" connectionString="data source=C:\Users\Test User\Desktop\newEPIWebAppSolution\EPIBiz\EPIBiz\app_data.db3;Version=3;Journal Mode=WAL;" providerName="System.Data.SQLite"/>
  </connectionStrings>
  <appSettings>
    <add key="aspnet:MaxHttpCollectionKeys" value="5000"/>
    <add key="EnableSSLOnMail" value="True"/>
    <add key="CrystalImageCleaner-AutoStart" value="true"/>
    <add key="CrystalImageCleaner-Sleep" value="60000"/>
    <add key="CrystalImageCleaner-Age" value="120000"/>
  </appSettings>
  <system.net>
    <mailSettings>

    </mailSettings>
  </system.net>
  <location path="~/Styles">
    <system.web>
      <authorization>
        <allow users="*"/>
      </authorization>
    </system.web>
  </location>
  <location path="WebResource.axd">
    <system.web>
      <authorization>
        <allow users="*"/>
      </authorization>
    </system.web>
  </location>
  <location path="ScriptResource.axd">
    <system.web>
      <authorization>
        <allow users="*"/>
      </authorization>
    </system.web>
  </location>
  <location path="Account" allowOverride="true">
    <system.web>
      <authorization>
        <allow users="*"/>
      </authorization>
    </system.web>
  </location>
  <location path="Docs" allowOverride="true">
    <system.web>
      <authorization>
        <deny users="*"/>
      </authorization>
    </system.web>
  </location>
  <location path="Images" allowOverride="true">
    <system.web>
      <authorization>
        <allow users="*"/>
      </authorization>
    </system.web>
  </location>
  <location path="ReportForm1.aspx">
    <system.web>
      <identity impersonate="true"/>
    </system.web>
  </location>
  <location path="Admin/SQLiteToExcel.aspx">
    <system.web>
      <httpRuntime executionTimeout="1000"/>
    </system.web>
  </location>
  <location path="Account/ResetPassword.aspx">
    <system.web>
      <authorization>
        <allow users="*"/>
      </authorization>
    </system.web>
  </location>
  <system.webServer>
    <validation validateIntegratedModeConfiguration="false"/>
    <defaultDocument enabled="true">
      <files>
        <clear/>
        <add value="Account/Login.aspx"/>
      </files>
    </defaultDocument>
    <modules>
      <add name="ScriptModule" preCondition="integratedMode" type="System.Web.Handlers.ScriptModule, System.Web.Extensions,Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35"/>
      <add type="DevExpress.Web.ASPxHttpHandlerModule, DevExpress.Web.v14.2, Version=14.2.13.0, Culture=neutral, PublicKeyToken=b88d1754d700e49a" name="ASPxHttpHandlerModule"/>
    </modules>
    <handlers>
      <remove name="WebServiceHandlerFactory-Integrated"/>
      <!--<remove name="ScriptHandlerFactory"/>
      <remove name="ScriptHandlerFactoryAppServices"/>
      <remove name="ScriptResource"/>-->

      <add name="ScriptHandlerFactory" verb="*" path="*.asmx" preCondition="integratedMode" type="System.Web.Script.Services.ScriptHandlerFactory, System.Web.Extensions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35"/>
      <add name="ScriptHandlerFactoryAppServices" verb="*" path="*_AppService.axd" preCondition="integratedMode" type="System.Web.Script.Services.ScriptHandlerFactory, System.Web.Extensions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35"/>
      <add name="ScriptResource" preCondition="integratedMode" verb="GET,HEAD" path="ScriptResource.axd" type="System.Web.Handlers.ScriptResourceHandler, System.Web.Extensions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35"/>
      <add type="DevExpress.Web.ASPxHttpHandlerModule, DevExpress.Web.v14.2, Version=14.2.13.0, Culture=neutral, PublicKeyToken=b88d1754d700e49a" verb="GET,POST" path="DX.ashx" name="ASPxHttpHandlerModule" preCondition="integratedMode"/>
    </handlers>
  </system.webServer>
  <!--
    For a description of web.config changes see http://go.microsoft.com/fwlink/?LinkId=235367.

    The following attributes can be set on the <httpRuntime> tag.
      <system.Web>
        <httpRuntime targetFramework="4.5" />
      </system.Web>
  -->
  <system.web>
    <customErrors mode="Off"/>
    <trace enabled="false" pageOutput="false" requestLimit="40" localOnly="false"/>
    <authentication mode="Forms">
      <forms loginUrl="~/Account/Login.aspx" path="/" timeout="30"/>
    </authentication>
    <httpHandlers>
      <remove verb="*" path="*.asmx"/>
      <add path="ScriptResource.axd" verb="GET,HEAD" type="System.Web.Handlers.ScriptResourceHandler, System.Web.Extensions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35" validate="false"/>
      <add verb="*" path="*.asmx" validate="false" type="System.Web.Script.Services.ScriptHandlerFactory, System.Web.Extensions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35"/>
      <add verb="*" path="*_AppService.axd" validate="false" type="System.Web.Script.Services.ScriptHandlerFactory, System.Web.Extensions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35"/>
      <add type="DevExpress.Web.ASPxHttpHandlerModule, DevExpress.Web.v14.2, Version=14.2.13.0, Culture=neutral, PublicKeyToken=b88d1754d700e49a" verb="GET,POST" path="DX.ashx" validate="false"/>
    </httpHandlers>
    <httpModules>
      <add name="ScriptModule" type="System.Web.Handlers.ScriptModule, System.Web.Extensions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35"/>
      <add name="ASPxHttpHandlerModule" type="DevExpress.Web.ASPxHttpHandlerModule, DevExpress.Web.v14.2, Version=14.2.13.0, Culture=neutral, PublicKeyToken=b88d1754d700e49a"/>
      <add type="DevExpress.Web.ASPxHttpHandlerModule, DevExpress.Web.v14.2, Version=14.2.13.0, Culture=neutral, PublicKeyToken=b88d1754d700e49a" name="ASPxHttpHandlerModule"/>
    </httpModules>
    <pages enableEventValidation="false" maintainScrollPositionOnPostBack="true" controlRenderingCompatibilityVersion="4.0">
      <controls>
        <add tagPrefix="asp" namespace="System.Web.UI" assembly="System.Web.Extensions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35"/>
        <!--<add tagPrefix="ajaxToolkit" assembly="AjaxControlToolkit" namespace="AjaxControlToolkit" />-->
      </controls>
    </pages>
    <siteMap defaultProvider="XmlSiteMapProvider" enabled="true">
      <providers>
        <add name="XmlSiteMapProvider" type="System.Web.XmlSiteMapProvider " siteMapFile="Web.sitemap" securityTrimmingEnabled="true"/>
        <remove name="MySqlSiteMapProvider"/>
      </providers>
    </siteMap>
    <compilation debug="true" targetFramework="4.5.1" defaultLanguage="c#">
      <assemblies>
        <!--<add assembly="Microsoft.Web.Infrastructure, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35"/>-->
        <!--<add assembly="System.Web.Extensions.Design, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35" />-->
        <add assembly="System.Web.Extensions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35"/>
        <add assembly="System.Web.Extensions.Design, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35"/>
        <add assembly="System.Management, Version=4.0.0.0, Culture=neutral, PublicKeyToken=B03F5F7F11D50A3A"/>
        <add assembly="System.Design, Version=4.0.0.0, Culture=neutral, PublicKeyToken=B03F5F7F11D50A3A"/>
        <add assembly="System.Web.Abstractions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35"/>
        <add assembly="System.Security, Version=4.0.0.0, Culture=neutral, PublicKeyToken=B03F5F7F11D50A3A"/>
        <add assembly="System.Web.Entity, Version=4.0.0.0, Culture=neutral, PublicKeyToken=B77A5C561934E089"/>
        <add assembly="System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=B77A5C561934E089"/>
        <add assembly="System.Xml.Linq, Version=4.0.0.0, Culture=neutral, PublicKeyToken=B77A5C561934E089"/>
        <add assembly="System.Data.Services.Client, Version=4.0.0.0, Culture=neutral, PublicKeyToken=B77A5C561934E089"/>
        <add assembly="Microsoft.Build.Framework, Version=4.0.0.0, Culture=neutral, PublicKeyToken=B03F5F7F11D50A3A"/>
        <add assembly="WindowsBase, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35"/>
        <add assembly="Microsoft.Exchange.WebServices, Version=15.0.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35"/>
        <add assembly="DevExpress.Web.ASPxThemes.v14.2, Version=14.2.13.0, Culture=neutral, PublicKeyToken=b88d1754d700e49a"/>
        <add assembly="DevExpress.XtraReports.v14.2.Web, Version=14.2.13.0, Culture=neutral, PublicKeyToken=b88d1754d700e49a"/>
        <add assembly="DevExpress.XtraReports.v14.2, Version=14.2.13.0, Culture=neutral, PublicKeyToken=b88d1754d700e49a"/>
        <add assembly="DevExpress.Data.v14.2, Version=14.2.13.0, Culture=Neutral, PublicKeyToken=b88d1754d700e49a"/>
        <add assembly="DevExpress.Web.ASPxHtmlEditor.v14.2, Version=14.2.13.0, Culture=neutral, PublicKeyToken=B88D1754D700E49A"/>
        <add assembly="DevExpress.Web.ASPxSpellChecker.v14.2, Version=14.2.13.0, Culture=neutral, PublicKeyToken=B88D1754D700E49A"/>
        <add assembly="DevExpress.SpellChecker.v14.2.Core, Version=14.2.13.0, Culture=neutral, PublicKeyToken=B88D1754D700E49A"/>
      </assemblies>
      <buildProviders>
        <add extension=".edmx" type="System.Data.Entity.Design.AspNet.EntityDesignerBuildProvider"/>
      </buildProviders>
    </compilation>
    <!--Membership Provider Configuration-->
    <membership defaultProvider="SQLiteMembershipProvider" hashAlgorithmType="SHA1" userIsOnlineTimeWindow="10">
      <providers>
        <clear/>
        <add applicationName="SQLite ASP.NET Provider" passwordFormat="Hashed" minRequiredNonalphanumericCharacters="1" minRequiredPasswordLength="6" maxInvalidPasswordAttempts="5" enablePasswordReset="true" passwordAttemptWindow="10" requiresQuestionAndAnswer="false" requiresUniqueEmail="true" connectionStringName="UserDataConnection" enablePasswordRetrieval="false" name="SQLiteMembershipProvider" type="TechInfoSystems.Data.SQLite.SQLiteMembershipProvider, TechInfoSystems.Data.SQLiteProvider"/>
      </providers>
    </membership>
    <!--Roll Provider Configuration-->
    <roleManager enabled="true" cacheRolesInCookie="true" cookieProtection="Validation" cookieRequireSSL="true" createPersistentCookie="false" defaultProvider="SQLiteRoleProvider">
      <providers>
        <clear/>
        <add applicationName="SQLite ASP.NET Provider" connectionStringName="UserDataConnection" name="SQLiteRoleProvider" type="TechInfoSystems.Data.SQLite.SQLiteRoleProvider, TechInfoSystems.Data.SQLiteProvider"/>
      </providers>
    </roleManager>
    <!--Profile Provider Configuration-->
    <profile enabled="true" defaultProvider="SQLiteProfileProvider">
      <providers>
        <clear/>
        <add applicationName="SQLite ASP.NET Provider" connectionStringName="UserDataConnection" name="SQLiteProfileProvider" type="TechInfoSystems.Data.SQLite.SQLiteProfileProvider, TechInfoSystems.Data.SQLiteProvider"/>
      </providers>
      <properties>
        <add name="GetSupervisor" type="string" serializeAs="String"/>
        <add name="ActivationKey" type="string" serializeAs="String"/>
      </properties>
    </profile>
  </system.web>
  <system.web.extensions>
    <scripting>
      <scriptResourceHandler enableCompression="false" enableCaching="false"/>
      <!-- Uncomment this line to customize maxJsonLength and add a custom converter -->
      <!--
<jsonSerialization maxJsonLength="500">
<converters>
<add name="ConvertMe" type="Acme.SubAcme.ConvertMeTypeConverter"/>
</converters>
</jsonSerialization>
-->
      <!-- Uncomment this line to enable the authentication service. Include requireSSL="true" if appropriate. -->
      <!--
<authenticationService enabled="true" requireSSL = "true|false"/>
-->
      <!-- Uncomment these lines to enable the profile service. To allow profile properties to be retrieved
and modified in ASP.NET AJAX applications, you need to add each property name to the readAccessProperties and
writeAccessProperties attributes. -->
      <!--
<profileService enabled="true"
readAccessProperties="propertyname1,propertyname2"
writeAccessProperties="propertyname1,propertyname2" />
-->
    </scripting>
  </system.web.extensions>
  <system.data>
    <DbProviderFactories>
      <remove invariant="System.Data.SQLite.EF6"/>
      <add name="SQLite Data Provider (Entity Framework 6)" invariant="System.Data.SQLite.EF6" description=".NET Framework Data Provider for SQLite (Entity Framework 6)" type="System.Data.SQLite.EF6.SQLiteProviderFactory, System.Data.SQLite.EF6"/>
    </DbProviderFactories>
  </system.data>
  <doddleReport defaultWriter="Html" dataRowStyle="DataRowStyle" headerRowStyle="HeaderRowStyle" footerRowStyle="FooterRowStyle">
    <writers>
      <add format="ExcelOpenXml" type="DoddleReport.OpenXml.ExcelReportWriter, DoddleReport.OpenXml" contentType="application/vnd.openxmlformats-officedocument.spreadsheetml.sheet" offerDownload="true" fileExtension=".xlsx"/>
      <add format="AbcPdf" type="DoddleReport.AbcPdf.PdfReportWriter, DoddleReport.AbcPdf" contentType="application/pdf" offerDownload="false" fileExtension=".pdf"/>
      <clear/>
      <add format="Html" type="DoddleReport.Writers.HtmlReportWriter, DoddleReport" contentType="text/html;charset=UTF-8" fileExtension=".html"/>
      <add format="Text" type="DoddleReport.Writers.DelimitedTextReportWriter, DoddleReport" contentType="text/plain;charset=UTF-8" fileExtension=".txt" offerDownload="true"/>
      <add format="Excel" type="DoddleReport.Writers.ExcelReportWriter, DoddleReport" contentType="application/vnd.ms-excel" offerDownload="true" fileExtension=".xls"/>
      <add format="iTextSharpPdf" type="DoddleReport.iTextSharp.PdfReportWriter, DoddleReport.iTextSharp" contentType="application/pdf" offerDownload="false" fileExtension=".pdf"/>
    </writers>
    <styles>
      <style name="DataRowStyle"/>
      <style name="HeaderRowStyle" bold="true" underline="true"/>
      <style name="FooterRowStyle" bold="true"/>
      <style name="Footer" italic="true"/>
      <style name="Title" fontSize="16"/>
    </styles>
  </doddleReport>
  <devExpress>
    <themes enableThemesAssembly="true" styleSheetTheme="" theme="BlackGlass" customThemeAssemblies=""/>
    <!--<compression enableHtmlCompression="false" enableCallbackCompression="true" enableResourceCompression="true" enableResourceMerging="true"/>-->
    <settings doctypeMode="Xhtml" rightToLeft="false" embedRequiredClientLibraries="false" ieCompatibilityVersion="edge"/>
    <errors callbackErrorRedirectUrl=""/>
    <compression enableHtmlCompression="false" enableCallbackCompression="true" enableResourceCompression="true" enableResourceMerging="true"/>
  </devExpress>
  <runtime>
    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
      <dependentAssembly>
        <assemblyIdentity name="System.Data.SQLite" publicKeyToken="db937bc2d44ff139" culture="neutral"/>
        <bindingRedirect oldVersion="0.0.0.0-1.0.90.0" newVersion="1.0.90.0"/>
      </dependentAssembly>
      <dependentAssembly>
        <assemblyIdentity name="System" publicKeyToken="b77a5c561934e089" culture="neutral"/>
        <bindingRedirect oldVersion="0.0.0.0-4.0.0.0" newVersion="4.0.0.0"/>
      </dependentAssembly>
      <dependentAssembly>
        <assemblyIdentity name="System.Drawing" publicKeyToken="b03f5f7f11d50a3a" culture="neutral"/>
        <bindingRedirect oldVersion="0.0.0.0-4.0.0.0" newVersion="4.0.0.0"/>
      </dependentAssembly>
      <dependentAssembly>
        <assemblyIdentity name="System.Data" publicKeyToken="b77a5c561934e089" culture="neutral"/>
        <bindingRedirect oldVersion="0.0.0.0-4.0.0.0" newVersion="4.0.0.0"/>
      </dependentAssembly>
      <dependentAssembly>
        <assemblyIdentity name="System.Windows.Forms" publicKeyToken="b77a5c561934e089" culture="neutral"/>
        <bindingRedirect oldVersion="0.0.0.0-4.0.0.0" newVersion="4.0.0.0"/>
      </dependentAssembly>
      <dependentAssembly>
        <assemblyIdentity name="System.Web" publicKeyToken="b03f5f7f11d50a3a" culture="neutral"/>
        <bindingRedirect oldVersion="0.0.0.0-4.0.0.0" newVersion="4.0.0.0"/>
      </dependentAssembly>
      <dependentAssembly>
        <assemblyIdentity name="System.Xml" publicKeyToken="b77a5c561934e089" culture="neutral"/>
        <bindingRedirect oldVersion="0.0.0.0-4.0.0.0" newVersion="4.0.0.0"/>
      </dependentAssembly>
      <dependentAssembly>
        <assemblyIdentity name="System.Web.Services" publicKeyToken="b03f5f7f11d50a3a" culture="neutral"/>
        <bindingRedirect oldVersion="0.0.0.0-4.0.0.0" newVersion="4.0.0.0"/>
      </dependentAssembly>
      <dependentAssembly>
        <assemblyIdentity name="System.Configuration" publicKeyToken="b03f5f7f11d50a3a" culture="neutral"/>
        <bindingRedirect oldVersion="0.0.0.0-4.0.0.0" newVersion="4.0.0.0"/>
      </dependentAssembly>
      <dependentAssembly>
        <assemblyIdentity name="System.ComponentModel.DataAnnotations" publicKeyToken="31bf3856ad364e35" culture="neutral"/>
        <bindingRedirect oldVersion="0.0.0.0-4.0.0.0" newVersion="4.0.0.0"/>
      </dependentAssembly>
      <dependentAssembly>
        <assemblyIdentity name="System.Web.Abstractions" publicKeyToken="31bf3856ad364e35" culture="neutral"/>
        <bindingRedirect oldVersion="0.0.0.0-4.0.0.0" newVersion="4.0.0.0"/>
      </dependentAssembly>
      <dependentAssembly>
        <assemblyIdentity name="WindowsBase" publicKeyToken="31bf3856ad364e35" culture="neutral"/>
        <bindingRedirect oldVersion="0.0.0.0-4.0.0.0" newVersion="4.0.0.0"/>
      </dependentAssembly>
      <dependentAssembly>
        <assemblyIdentity name="System.Security" publicKeyToken="b03f5f7f11d50a3a" culture="neutral"/>
        <bindingRedirect oldVersion="0.0.0.0-4.0.0.0" newVersion="4.0.0.0"/>
      </dependentAssembly>
      <dependentAssembly>
        <assemblyIdentity name="Microsoft.Build.Framework" publicKeyToken="b03f5f7f11d50a3a" culture="neutral"/>
        <bindingRedirect oldVersion="0.0.0.0-4.0.0.0" newVersion="4.0.0.0"/>
      </dependentAssembly>
      <dependentAssembly>
        <assemblyIdentity name="System.Design" publicKeyToken="b03f5f7f11d50a3a" culture="neutral"/>
        <bindingRedirect oldVersion="0.0.0.0-4.0.0.0" newVersion="4.0.0.0"/>
      </dependentAssembly>
      <dependentAssembly>
        <assemblyIdentity name="Antlr3.Runtime" publicKeyToken="eb42632606e9261f" culture="neutral"/>
        <bindingRedirect oldVersion="0.0.0.0-3.5.0.2" newVersion="3.5.0.2"/>
      </dependentAssembly>
      <dependentAssembly>
        <assemblyIdentity name="WebGrease" publicKeyToken="31bf3856ad364e35" culture="neutral"/>
        <bindingRedirect oldVersion="0.0.0.0-1.5.2.14234" newVersion="1.5.2.14234"/>
      </dependentAssembly>
      <dependentAssembly>
        <assemblyIdentity name="itextsharp" publicKeyToken="8354ae6d2174ddca" culture="neutral"/>
        <bindingRedirect oldVersion="0.0.0.0-5.1.1.0" newVersion="5.1.1.0"/>
      </dependentAssembly>
      <dependentAssembly>
        <assemblyIdentity name="System.Xml.Linq" publicKeyToken="b77a5c561934e089" culture="neutral"/>
        <bindingRedirect oldVersion="0.0.0.0-4.0.0.0" newVersion="4.0.0.0"/>
      </dependentAssembly>
      <dependentAssembly>
        <assemblyIdentity name="System.Web.Extensions" publicKeyToken="31bf3856ad364e35" culture="neutral"/>
        <bindingRedirect oldVersion="0.0.0.0-4.0.0.0" newVersion="4.0.0.0"/>
      </dependentAssembly>
      <dependentAssembly>
        <assemblyIdentity name="System.Web.Extensions.Design" publicKeyToken="31bf3856ad364e35" culture="neutral"/>
        <bindingRedirect oldVersion="0.0.0.0-4.0.0.0" newVersion="4.0.0.0"/>
      </dependentAssembly>
    </assemblyBinding>
  </runtime>
  <entityFramework>
    <defaultConnectionFactory type="System.Data.Entity.Infrastructure.LocalDbConnectionFactory, EntityFramework">
      <parameters>
        <parameter value="v12.0"/>
      </parameters>
    </defaultConnectionFactory>
    <providers>
      <provider invariantName="System.Data.SqlClient" type="System.Data.Entity.SqlServer.SqlProviderServices, EntityFramework.SqlServer"/>
      <provider invariantName="System.Data.SQLite.EF6" type="System.Data.SQLite.EF6.SQLiteProviderServices, System.Data.SQLite.EF6"/>
    </providers>
  </entityFramework>
</configuration>
```
Any assistance would be great.

**Comment by AlekseyMartynov** [comment]
From code snippets, I can see that `ScriptManager` has `LoadScriptsBeforeUI="False"`. Please try my suggestion from https://github.com/DevExpress/AjaxControlToolkit/issues/152#issuecomment-231026249. Does it help in this case?

See also: [Sys.Net.WebServiceProxy.invoke doesn't work](http://forums.asp.net/t/2038887.aspx)

**Comment by eldridgegl** [comment]
I have now done a few things. I first downgraded to AjaxControlToolkit version 7.1.
I removed AjaxFrameworkMode="Explicit" from toolkitscriptmanager since that was a workaround attempt to get things going. It seems that converting from website to web application, there must be a bug with runtime scriptmanager.axd since it was failing on extenders. The solution to the overall problem was to change CombineScripts="False" even though it was True with website. Setting to False has gotten my extenders to work.
